ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -191,6 +192,9 @@
     <Compile Include="ZipStreamProvider.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="AccernDataSplitter\config.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Typically we set this to copy if newer so if you modify the `config.json` in the bin directory it doesn't get overwritten with each build.